applications/serial_lte_modem/src/main.c

```diff
@@ -291,14 +284,10 @@ static void handle_nrf_modem_lib_init_ret(int ret)
 		fota_info = ret;
 		break;
 	}
-
-	slm_setting_fota_save();
```

Comment:
Also, it is not guaranteed that we can run until slm_settings_fota_save is called. Depending how fatal it would be if that happened, it may make sense to still store settings in here.